docs/preview/features/databricks/databricks-info-provider.md

```diff
@@ -0,0 +1,47 @@
+### Databricks information provider
+
+The background job makes use of the available `DatabricksInfoProvider` to measure and report the finished job outcomes.
```

Comment:
We should tell them how to install
```suggestion
## Installation

To use these features, you have to install the following package:

	```shell
	PM > Install-Package Arcus.BackgroundJobs.Databricks
	```

## Usage
The background job makes use of the available `DatabricksInfoProvider` to measure and report the finished job outcomes.
```